src/site/content/en/lighthouse-performance/lighthouse-max-potential-fid/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Max Potential First Input Delay
+description: |
+  Learn about Lighthouse's Max Potential First Input Delay metric and
+  how to measure and optimize it.
+date: 2019-05-02
+updated: 2019-10-16
+web_lighthouse:
+  - max-potential-fid
+---
```

Comment:
@robdodson I renamed this from `/max-potential-fid` to `/lighthouse-max-potential-fid` to avoid naming conflicts with Philip's impending canonical metrics documentation. Because Lighthouse doesn't yet link to this doc I don't think it's a big deal to change the URL. LMK if you agree.

Reply:
I think that's ok... kind of bummer that we have 1 metric doc already squatting on a name and all the LH docs squatting on the others.